sinch/domains/sms/endpoints/batches/send_batch_dry_run.py

```diff
@@ -23,7 +23,7 @@ def build_url(self, sinch) -> str:
 
     def build_query_params(self):
         return {
-            "per_recipient": "true" if self.request_data.per_recipient else "false",
+            "per_recipient": str(self.request_data.per_recipient).lower(),
```

Comment:
This is reverting the commit https://github.com/sinch/sinch-sdk-python/pull/3/commits/63bd2d35ef29f536c8b6affe9417801958e5053e
Is this intended ?

Reply:
@JPPortier nah, it's calling `.lower()` additionally